content/blog/scheduling-in-react/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Scheduling in React"
+date: "2019-03-15T12:00:00.000Z"
+---
+
+In modern applications, user interfaces often have to juggle multiple tasks at the same time. For example, a search component needs to respond to user input while providing auto completion results, and an interactive dashboard needs to update charts while loading data from the server and sending analytics data to your backend.
```

Comment:
Does it "need" to do something or can we say "might need to?" This comes down to a question of Are these the only tasks these things do, or no? 

Also, can we say "a backend" instead of "your?" Or are you specifying that for a reason?

"For example, a search component needs to respond to user input while providing auto completion results, and an interactive dashboard needs to update charts while loading data from the server and sending analytics data to your backend."

Reply:
> Does it "need" to do something or can we say "might need to?" This comes down to a question of Are these the only tasks these things do, or no?

Might need to, you're right.

> Also, can we say "a backend" instead of "your?" Or are you specifying that for a reason?

a backend seems better, yes! :-)